test/images/logs-generator/Makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TAG = v1.0.0
```

Comment:
I just don't see how it can be "released" to the first version, this tool doesn't have any development lifecycle, just one release and mb occasional updates

So why not use discrete major versions to avoid multiplying entities?